lib/bootsnap/cache_wrapper.rb

```diff
@@ -0,0 +1,55 @@
+# Wrapper to accept a number of common cache implementations.
+module Bootsnap
+  class CacheWrapper
+    def self.get(cache)
+      # `Cache#get(key)` for Memcache
+      if cache.respond_to?(:get)
```

Comment:
I really dislike this pattern, the cache bootsnap takes in should always require a consistent interface not a "anything goes" one, otherwise it is #respond_to central for zero gain.